desub/src/lib.rs

```diff
@@ -82,9 +83,13 @@ impl<T: TypeDetective> Decoder<T> {
 	}
 }
 
+/// Decoder which does not resolve any types of metadata v13 and below.
+/// To be used with newer chains where legacy types are not required.
 pub struct InfoDecoder(Decoder<NoLegacyTypes>);
-impl Default for InfoDecoder {
-	fn default() -> Self {
+impl InfoDecoder {
+    // No sensible default b/c of Chain::Custom("none")
```

Comment:
funny whitespace here?

Reply:
my rust version was pre-2021 and i didn't even realize the linter couldn't run :(

all fixed